packages/gatsby-cli/src/init-starter.js

```diff
@@ -67,8 +68,29 @@ const install = async rootPath => {
   process.chdir(rootPath)
 
   try {
-    let cmd = shouldUseYarn() ? spawn(`yarnpkg`) : spawn(`npm install`)
-    await cmd
+    const npmCmd = `npm install`
+    let response = npmCmd
+    if (shouldUseYarn()) {
```

Comment:
So in this it'll only prompt the user is `yarn` is detected and if not, it'll continue with a prompt. Correct?

Reply:
Sure !

Reply:
This is great! no need to ask if they don't have it installed.